### DISCUSSION TITLE
Deluge Tags(labels)

### DISCUSSION THREAD

**Discussion by xtinct101** [jesec/flood#607] [General]
Hi,

I know Deluge support is still listed as experimental and tags was listed as not supported.
Is this still something that's planned on being added?

**Comment by dn4hc** [comment]
Would be really useful.

**Comment by LordFransie** [comment]
I'll third this. There is a terrible UI named storm and the only reason I keep it running is incase I need to modify a label. I'd love if flood let us do this.

**Comment by strike84** [comment]
+1 Would love to see this too
